src/sentry/api/endpoints/organization_events_trace.py

```diff
@@ -1063,11 +1096,23 @@ def get(self, request: HttpRequest, organization: Organization, trace_id: str) -
                 warning_extra,
                 event_id,
                 detailed,
-                tracing_without_performance_enabled,
                 use_spans,
             )
         )
 
+    def serialize(
+        self,
+        limit: int,
+        transactions: Sequence[SnubaTransaction],
+        errors: Sequence[SnubaError],
+        roots: Sequence[SnubaTransaction],
+        warning_extra: dict[str, str],
+        event_id: str | None,
+        detailed: bool = False,
+        use_spans: bool = False,
+    ) -> Any:
+        pass
```

Comment:
Is this an abstract method that is overridden in the child classes? If so, can it

```python
raise NotImplementedError
```

Bonus points if you turn this into an abstract class and decorate this with `@abc.abstractmethod`

Reply:
fiiiine